src/webui/gitutils.jl

```diff
@@ -118,14 +137,32 @@ function make_registration_request(
     title::AbstractString,
     body::AbstractString,
 )
-    return create_pull_request(
-        r.forge, r.repo.id;
+    repoid = r.repo.id
+    base = r.repo.default_branch
+    pr = create_pull_request(
+        r.forge, repoid;
         source_branch=branch,
-        target_branch=r.repo.default_branch,
+        target_branch=base,
         title=title,
         description=body,
         remove_source_branch=true,
     )
+    pr.ex === nothing && return pr
+    if pr.resp === nothing || pr.resp.status != 409
```

Comment:
That still throws an exception. Did you mean to return the result?

Reply:
I'm just replicating your existing code, ideally it would do something else. I think this function could just return a `Bool` representing whether or not the result contains a "already exists" error, and if it doesn't, then the result can just be immediately returned to the `make_registration_request` caller. That would be interpreted as "Making pull request failed", which makes sense since there is some other unexpected error.